ts/features/wallet/onboarding/bancomat/saga/networking/index.ts

```diff
@@ -31,6 +31,8 @@ export function* handleLoadAbi(
     }
   } catch (e) {
     yield put(loadAbi.failure(e));
+    yield delay(200);
```

Comment:
Is this a code for tests?

Reply:
I preferred to keep a minimum delay before making the API request restart in order to avoid an excessive call to the PM API, what do you think?